views/admin/layout.jade

```diff
@@ -21,7 +22,25 @@ html(lang='en')
         min-width: 100%;
         *width: 100%;
       }
-  body
+      .rightSideBar{
```

Comment:
replace this with popout. popout should be the button and .rightsidebar should be the div